### ISSUE TITLE
Discord and Slack crash if a gnome notification of theirs is clicked

### ISSUE THREAD

**Issue by Sciencentistguy** [NixOS/nixpkgs#173783]
### Describe the bug
Discord and Slack crash if a gnome notification of theirs is clicked

### Steps To Reproduce
1. Have discord or slack open in GNOME
2. Receive a message that generates a notification
3. Click it
4. ...
5. Crash!

### Expected behavior
The application should navigate to the correct page from the notification (like it used to), not crash.

### Additional context
This happens on my NixOS system (chronos) specified by this flake: https://github.com/Sciencentistguy/nixfiles/tree/95e28f80e7df5b8c01e376d9305ad56b4895e974

This is not the first commit that exhibits this bug, but I do not currently have the time to bisect for it.

### Notify maintainers
GNOME:
- @NixOS/gnome 

Discord:
- @ldesgoui 
- @MP2E 
- @devins2518 

Slack:
- @mmahut 
- @maxeaubrey 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.39, NixOS, 22.05 (Quokka), 22.05.20220518.52dc75a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - channels(root): `"home-manager, nixos"`
 - nixpkgs: `/etc/nix/path/nixpkgs`
```

**Comment by jtojnar** [comment]
Could you get stack trace by running `coredumpctl gdb` after the crash and entering `bt full` there?

**Comment by LunNova** [comment]
Fair chance it's this fixed libnotify issue: https://gitlab.gnome.org/GNOME/libnotify/-/merge_requests/26 which was merged in #172287 and is now in the unstable branches.
